app/src/main/java/com/infomaniak/mail/data/models/message/Message.kt

```diff
@@ -85,24 +88,30 @@ class Message : RealmObject {
     @SerialName("is_duplicate")
     var isDuplicate: Boolean = false
     var duplicates: RealmList<Duplicate> = realmListOf()
+    //endregion
 
-    /**
-     * Local
-     */
-    @Ignore
+    //region Local data (Transient)
+    @Transient
     var draftUuid: String? = null
-    @Ignore
+    @Transient
     var fullyDownloaded: Boolean = false
-    @Ignore
+    @Transient
     var hasUnsubscribeLink: Boolean = false
-    @Ignore
+    @Transient
     var parentLink: Thread? = null // TODO
```

Comment:
You have to use an inversion relation for that

Reply:
Impossible 😭 

https://github.com/realm/realm-kotlin/issues/591